tests/TestDbEntity.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+
+namespace Starlit\Db;
```

Comment:
Not sure about this namespace here?
Maybe it should also be moved to say `tests/resources/TestDbEntity.php` or something like that instead?

Reply:
I agree. Maybe we should think about using dedicated namespaces for the tests in general, e.g. `Starlit\Db\Tests` ?